lib/compat/wordpress-6.1/class-wp-theme-json-data.php

```diff
@@ -0,0 +1,60 @@
+<?php
+/**
+ * API to update a theme.json structure.
+ *
+ * @package gutenberg
+ */
+
+/**
+ * Class to update with a theme.json structure.
+ */
+class WP_Theme_JSON_Data {
+
+	/**
+	 * Container of the data to update.
+	 *
+	 * @var WP_Theme_JSON
+	 */
+	private $theme_json = null;
+
+	/**
+	 * The origin of the data: default, theme, user, etc.
+	 *
+	 * @var string
+	 */
+	private $origin = null;
```

Comment:
This should probably default to `''` (empty string) if the type is explicitly set as string.